pages/tech.js

```diff
@@ -78,8 +78,7 @@ const tech = () => (
     </div>
     <div className="section-2">
       <div className="hero">
-        <Picture url={prefixLink('/img/Berkeley 1.jpg')} />
-        <Title text="PILOT PLANT" />
+        <Title text="PILOT PLANT:" />
       </div>
       <div className="section-container">
```

Comment:
For the class names that are block-element (i think of modifiers as numbers) please use the format `block__element` so in this case do `section__container`.  Here and elsewhere